docs/source/releases/index.rst

```diff
@@ -8,14 +8,6 @@
 Release Notes
 *************
 
-Latest (version on cutting edge of git)
----------------------------------------
-
-.. toctree::
-   :maxdepth: 1
-
-   latest
-
```

Comment:
No longer always including this in index.rst, because it will now be automaticaly added from build_docs.sh